presto-hive/src/main/java/com/facebook/presto/hive/metastore/UserTableKey.java

```diff
@@ -37,7 +37,7 @@ public UserTableKey(
             @JsonProperty("database") String database,
             @JsonProperty("table") String table)
     {
-        this.principal = requireNonNull(principal, "principal is null");
+        this.principal = principal;
```

Comment:
there is a comment in https://github.com/prestosql/presto/pull/90/commits/0cf6e5b0a3a0365e4bcfad6d0413fc1a7b8cdb65:
```
        // principal can be null when we want to list all privileges for admins
```